CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+# Unreleased
+
+## Added
+
+- We have a new media-query available in `$bitstyles-breakpoints`: `motion-ok`. It’s available using the `media-query` mixin, and applies when the user has not set a preference to reduce motion in their UIs, allowing a no-motion-first approach to animations
```

Comment:
**TIL** about `no-motion-first` 💡 

Neat!

Can you add a link for reference if you have a good one at hand?

Reply:
Yes! I’ll add it (it’s from the amazing Tatiana Mac at https://tatianamac.com/posts/prefers-reduced-motion/)

Reply:
That was also the link I found 😉